File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-print-documents_2.cs
            object copies = "1"; 
            object pages = "1"; 
            object range = Word.WdPrintOutRange.wdPrintCurrentPage; 
            object items = Word.WdPrintOutItem.wdPrintDocumentContent; 
            object pageType = Word.WdPrintOutPages.wdPrintAllPages; 
            object oTrue = true; 
            object oFalse = false; 

            this.PrintOut(
                ref oTrue, ref oFalse, ref range, ref missing, ref missing, ref missing,
                ref items, ref copies, ref pages, ref pageType, ref oFalse, ref oTrue, 
                ref missing, ref oFalse, ref missing, ref missing, ref missing, ref missing);

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-print-documents_2.vb
        Me.PrintOut( _
            Background:=True, _
            Append:=False, _
            Range:=Word.WdPrintOutRange.wdPrintCurrentPage, _
            Item:=Word.WdPrintOutItem.wdPrintDocumentContent, _
            Copies:="1", _
            Pages:="1", _
            PageType:=Word.WdPrintOutPages.wdPrintAllPages, _
            PrintToFile:=False, _
            Collate:=True, _
            ManualDuplexPrint:=False)